src/GayatriG/Assignment5Ex5.java

```diff
@@ -0,0 +1,43 @@
+//Write a Java program that keeps a number from the user and generates an integer between 1 and 7 
+//and displays the name of the weekday. (use switch case) 
+
+package GayatriG;
+
+import java.util.Scanner;
+
+public class Assignment5Ex5 {
+	static void displayWeekday(int input){
+		switch(input){
```

Comment:
If user wanted know day on 23rd date..
correct logic.
for that you have take 23%7